.github/test-spec.yml

```diff
@@ -188,6 +188,7 @@
   - "lib/date_time/**/*"
   - "drivers/sensor/sensor_sim/**/*"
   - "applications/asset_tracker_v2/**/*"
+  - "applications/asset_tracker_v2/tests/**/*"
```

Comment:
I wonder if we should just place this test at the same level as the other tests for now. The LwM2M codec test was placed there as well. We could change the folder structure afterwards.

Reply:
I moved the tests back for now.